gems/pending/Gemfile

```diff
@@ -36,7 +36,7 @@ gem "net-scp",                 "~>1.2.1",           :require => false
 gem "nokogiri",                "~>1.6.0",           :require => false
 gem "openshift_client",        "=1.1.0",            :require => false
 gem "openscap",                "~>0.4.3",           :require => false
-gem "ovirt",                   "~>0.7.2",           :require => false
+gem "ovirt",                   "~>0.7.3",           :require => false
```

Comment:
@pkliczewski change this to `0.8.0`

https://rubygems.org/gems/ovirt/versions/0.8.0